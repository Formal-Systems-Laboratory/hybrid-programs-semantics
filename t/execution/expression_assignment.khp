vars a ; a := 1.0 + (2.0 * 3.0)
