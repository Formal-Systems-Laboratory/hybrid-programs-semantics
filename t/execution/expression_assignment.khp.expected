<generatedTop>
  <k>
    .
  </k>
  <state>
    a |-> 1.0 + 2.0 * 3.0
  </state>
  <gPgmVars>
    a , .Ids
  </gPgmVars>
  <gNonDetAssignments>
    .Map
  </gNonDetAssignments>
  <gEvolutionConditions>
    .Set
  </gEvolutionConditions>
  <counter>
    0
  </counter>
  .SynthesisCell
</generatedTop>
